cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/DefaultSchedulerService.java

```diff
@@ -73,7 +73,11 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
         ProgramOptionConstants.SCHEDULE_NAME, scheduleName
       ));
 
-      taskRunner.run(Id.Program.from(accountId, applicationId, programId), programType, args);
+      try {
+        taskRunner.run(Id.Program.from(accountId, applicationId, programId), programType, args);
+      } catch (TaskExecutionException e) {
```

Comment:
Yes, there is. I want to use the ``ScheduleTaskRunner`` to trigger job execution in any schedule. The ``run`` method used to throw a quartz exception if the execution failed. It now throws a ``TaskExecutionException``, which all schedules can understand. The class in which this code is executed though, ``ScheduledJob``, implements a quartz interface which throws a ``JobExecutionException``.